gst/audioparsers/put_bits.h

```diff
@@ -223,6 +223,7 @@ static inline void put_sbits(PutBitContext *pb, int n, int32_t value)
 /**
  * Write exactly 32 bits into a bitstream.
  */
+#if 0
```

Comment:
Maybe defining av_unused to  [G_GNUC_UNSED ](https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-GNUC-UNUSED:CAPS) can help

Reply:
thanks, it worked ! Adding -Wall -Werror to compiler flags..

Reply:
oh, I was writing about a bit different issue - last warning.

Reply:
fixed